src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -3,6 +3,22 @@
 /**
  * Feel free to remove this class and create your own.
  */
+
 public class HelloWorld {
+    public static void main(String[] args) {
+        final int FigureCount = 6;
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure[] figures = new Figure[FigureCount];
+        for (int i = 0; i < figures.length; i++) {
+            if (i < figures.length / 2) {
+                figures[i] = figureSupplier.getRandomFigure();
+            } else {
+                figures[i] = figureSupplier.getDefaultFigure();
+            }
+        }
 
+        for (Figure figure : figures) {
+            System.out.println(figure);
+        }
```

Comment:
Do we need an additional loop to draw all figures? Could we draw them using the previous loop?